tests/ui/mul_add.rs

```diff
@@ -0,0 +1,29 @@
+#![warn(clippy::manual_mul_add)]
```

Comment:
```suggestion
// run-rustfix
#![warn(clippy::manual_mul_add)]
```

I'm not sure if this will work, because some of the suggestions have to be applied one after another. If that's the case, just split up the test in a `run-rustfix` file and a non-`run-rustfix` file.